database-mongodb/MySQL/Images.js

```diff
@@ -0,0 +1,66 @@
+const connection = require('./connect.js');
+const { promisifyAll } = require('bluebird');
+
+promisifyAll(connection);
+
+connection.connect((err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('Successfully connected to MySQL DB');
+  }
+});
+
+
+function insertRecords (records) {
+  let insertionsParsedToMySQLSyntax = '';
+  for (let i = 0; i < records.length; i++) {
+    if (i !== 999) {
+      insertionsParsedToMySQLSyntax = insertionsParsedToMySQLSyntax + "('" + records[i].itemId+ "','" + records[i].itemImages+ "'),";
```

Comment:
Maybe using template literals here? More of a suggestion, I love using template literals myself.

Reply:
The first iteration used template literals but then i had an issue wih MySQL not recognizing my strings as strings unless I literally wrapped them in single quoutes. While dong research into that issue since the error message from MySQL, of course, told me almost nothing, I came across something saying MySQL treats back ticks in a different manner. Most likely, those backticks won't be transferred from the JS context to MySQL's context but I decided to be safe and remove them. I can put it back to a template literal format and still see if works, if you want?

Reply:
No thats fine. I was having issues with postgres and quotes stuff too, makes sense.